submissions/evgeniy24/memory-pair-game/app.js

```diff
@@ -0,0 +1,89 @@
+const field = document.querySelector('.game');
+const cards = document.querySelectorAll('.card');
+const win = document.querySelector('.win');
+
+const  cardMaxPositions = 12;
+
+let isRolledCard = false,
+    lockRoll = false,
+    firstCard, secondCard,
+    winCondition = 6,
```

Comment:
It's better to make them constants in order to prevent reassignment in your further code

Reply:
I use const for winConditions and cardMaxPositions because they are not reassigned anywhere further! I use let for other variables because they are!